src/Generators/MigrationGenerator.php

```diff
@@ -53,18 +53,18 @@ public function output(array $tree): array
             if (!empty($model->pivotTables())) {
                 foreach ($model->pivotTables() as $pivotSegments) {
                     $pivotTable = $this->getPivotTableName($pivotSegments);
-                    if (isset($created_pivot_tables[$pivotTable])) {
```

Comment:
I'd rather leave this code in the loop and update the `isset` to `in_array`. This prevents a duplicate loop and is the simplest change.

Reply:
@jasonmccreary I tried that, but with `in_array` the migration order is incorret.

With `in_array` the migrations are:
```
[
    0 => "database/migrations/2020_05_01_191229_create_companies_table.php"
    1 => "database/migrations/2020_05_01_191229_create_company_person_table.php"
    2 => "database/migrations/2020_05_01_191230_create_people_table.php"
]
```

This way, migrations are:
```
[
    0 => "database/migrations/2020_05_01_191229_create_companies_table.php"
    1 => "database/migrations/2020_05_01_191230_create_people_table.php"
    2 => "database/migrations/2020_05_01_191230_create_company_person_table.php"
]
```

Reply:
I see. And the tests confirm the later timestamp, so I guess that's good.